planner/languages/rust/rust_planner.go

```diff
@@ -54,7 +58,20 @@ func (p *Planner) getPlan(srcDir string) (*plansdk.Plan, error) {
 
 	return &plansdk.Plan{
 		NixOverlays: []string{RustOxalicaOverlay},
-		DevPackages: []string{rustPkgDev},
+		// 'gcc' added as a linker for libc (C toolchain)
+		// 1. https://softwareengineering.stackexchange.com/a/332254
+		// 2. https://stackoverflow.com/a/56166959
+		DevPackages: []string{rustPkgDev, "gcc"},
+		// 'gcc' needs to be present to execute the binary
+		RuntimePackages: []string{"gcc"},
+		SharedPlan: plansdk.SharedPlan{
+			BuildStage: &plansdk.Stage{
+				Command: "cargo build --release",
```

Comment:
@savil when you rebase, I removed the default install stage `inputFiles` field, so you'd have to add
```
InstallStage: {
    InputFiles: []string{"."}
}
```